src/auth/containers/requireAdminAuth.tsx

```diff
@@ -1,50 +1,32 @@
 import React, { useEffect } from 'react';
```

Comment:
The changes in this, `requireAuth`, and `requireStoreAccess` will need to be re-applied to Paul's new access protection component. We'll want to merge from master to incorporate those changes.